core/src/main/java/org/apache/calcite/sql/dialect/MysqlSqlDialect.java

```diff
@@ -71,10 +71,22 @@ public class MysqlSqlDialect extends SqlDialect {
             return 65535;
           case TIMESTAMP:
             return 6;
+          case DECIMAL:
+            return 65;
           default:
             return super.getMaxPrecision(typeName);
           }
         }
+
+        @Override public int getMaxScale(SqlTypeName typeName) {
```

Comment:
I think this is wrong: https://dev.mysql.com/doc/refman/8.4/en/precision-math-decimal-characteristics.html
The docs say that 30 is the max value.
Please double check these values before you make the fixes.

Reply:
It‘s my mistake!  thanks for the correction，I would doube check them again. @mihaibudiu

Reply:
Had changed it after check from documents，and test precision and scale in Presto/MySQL/Phoenix real environment which as expected. PTAL @mihaibudiu

Reply:
ideally you can add here links to the documentation and specify also the version of the database, so people can check whether the values are still accurate as things change

Reply:
Done，had add document links of MySQL 5.x and 8.x(which are the most used version), thanks for the suggestion. @mihaibudiu